src/core/server/logging/layouts/json_layout.ts

```diff
@@ -46,20 +46,24 @@ export class JsonLayout implements Layout {
 
     return {
       message: error.message,
-      name: error.name,
-      stack: error.stack,
+      type: error.name,
+      stack_trace: error.stack,
     };
   }
 
   public format(record: LogRecord): string {
     return JSON.stringify({
       '@timestamp': moment(record.timestamp).format('YYYY-MM-DDTHH:mm:ss.SSSZ'),
-      context: record.context,
-      error: JsonLayout.errorToSerializableObject(record.error),
-      level: record.level.id.toUpperCase(),
       message: record.message,
-      meta: record.meta,
-      pid: record.pid,
+      error: JsonLayout.errorToSerializableObject(record.error),
+      log: {
+        level: record.level.id.toUpperCase(),
+        logger: record.context,
+      },
+      process: {
+        pid: record.pid,
+      },
+      ...record.meta,
```

Comment:
Is it possible to provide typings for ECS-compatible meta object?

Reply:
How detailed are you thinking?  e.g. The whole spec, base objects, or defaults we're setting?

Might be worth digging into https://github.com/elastic/ecs/issues/280